tools/targets/ARM_MUSCA_S1.py

```diff
@@ -19,13 +19,24 @@
 from os.path import abspath, basename, dirname, splitext, isdir
 from os.path import join as path_join
 import re
-from argparse import Namespace
+import sys
+from argparse import ArgumentParser, Namespace
+
+SCRIPT_DIR = dirname(abspath(__file__))
+MBED_OS_ROOT = abspath(path_join(SCRIPT_DIR, os.pardir, os.pardir))
+
+# Ensure the tools module is visible to this script
+sys.path.insert(0, MBED_OS_ROOT)
+
+TFM_SCRIPTS = abspath(path_join(dirname(__file__), '..', 'psa', 'tfm', 'bin_utils'))
+
+# Ensure the psa module is visible to this script
+sys.path.insert(0, TFM_SCRIPTS)
+
 from tools.psa.tfm.bin_utils.assemble import Assembly
```

Comment:
I am scared of this to see tools. there but that will be cleaned up hopefully via TFM support PR.